pkg/3scale/amp/component/system.go

```diff
@@ -47,6 +47,16 @@ const (
 	SystemSecretSystemRedisURLFieldName  = "URL"
 	SystemSecretSystemRedisSentinelHosts = "SENTINEL_HOSTS"
 	SystemSecretSystemRedisSentinelRole  = "SENTINEL_ROLE"
+
+	// TLS env vars
+	SystemSecretSystemRedisCAFile            = "REDIS_CA_FILE"
+	SystemSecretSystemRedisClientCertificate = "REDIS_CLIENT_CERT"
+	SystemSecretSystemRedisPrivateKey        = "REDIS_PRIVATE_KEY"
+	SystemSecretSystemRedisSSL               = "REDIS_SSL"
+
+	SystemSecretSystemRedisSslCa   = "SSL_CA"
+	SystemSecretSystemRedisSslCert = "SSL_CERT"
+	SystemSecretSystemRedisSslKey  = "SSL_KEY"
```

Comment:
@valerymo think we need to agree on a naming scheme here as it may be possible that a customer has different certs for Redis and the System-db. Propose that we prefix with `CACHE_` as may not be using redis going forward 


```suggestion
	SystemSecretSystemRedisSslCa   = "CACHE_SSL_CA"
	SystemSecretSystemRedisSslCert = "CACHE_SSL_CERT"
	SystemSecretSystemRedisSslKey  = "CACHE_SSL_KEY"
```
and I use  the prefix `SYSTEMDB_`
```suggestion
	SystemSecretSslCa                               = "SYSTEMDB_SSL_CA"
	SystemSecretSslCert                             = "SYSTEMDB_SSL_CERT"
	SystemSecretSslKey                              = "SYSTEMDB_SSL_KEY"
```
What do you think ?